src/Core/src/PropertyMapper.cs

```diff
@@ -31,6 +31,9 @@ protected virtual void SetPropertyCore(string key, Action<IElementHandler, IElem
 
 		protected virtual void UpdatePropertyCore(string key, IElementHandler viewHandler, IElement virtualView)
 		{
+			if (!viewHandler.CanUpdateProperty())
```

Comment:
Who is calling this if theres no view ?

Reply:
For example, AppThemeBinding invoke the method to apply a property. That will use the UpdateValue method from ElementHandler and a Mapper finally will invoke a MapProperty method. Inside that method we use the platform view but, that view could be disposed (GC fires and collects the Android view).